test/metabase/api/database_test.clj

```diff
@@ -269,26 +270,6 @@
 
 (def ^:private monthly-schedule {:schedule_type "monthly" :schedule_day "fri" :schedule_frame "last"})
 
-(defn- all-db-sync-triggers-name
```

Comment:
I moved these functions to `metabase.task.sync-databases-test` because they were being used in a few places. And since they mostly depend on that namespace it felt natural to put them there.